kedro/io/core.py

```diff
@@ -704,7 +704,7 @@ def get_protocol_and_path(filepath: str, version: Version = None) -> Tuple[str,
     protocol = options_dict["protocol"]
 
     if protocol in HTTP_PROTOCOLS:
-        if version:
+        if version is not None:
```

Comment:
Either way technically works. I did it for consistency, because I used `if version is not None` in the `MatplotibWriter`.

I think we also do explicitly mean to check `is not None` (that's what the docstring also says), and PEP-8 says, "Also, beware of writing if x when you really mean if x is not None." Even though a `Version` object can't evaluate to False, felt this may be more explicit/safer.